### PR TITLE
Remove deprecated RouteObjectInterface, PHP8.1 compatibility.

### PR DESCRIPTION
**Pull request by adubovskoy**

### CODE REVIEW

File: src/Codeception/Module/DrupalUser.php [hunk 1]
```diff
@@ -185,6 +187,7 @@ public function logInAs($username) {
    */
   public function logInWithRole($role) {
     $user = $this->createUserWithRoles([$role], Factory::create()->password(12, 14));
+    var_dump($user->getAccountName());
```
Comment (minorOffense) [inline src/Codeception/Module/DrupalUser.php]: Is this some debug left in?

File: src/Codeception/Module/DrupalUser.php [hunk 2]
```diff
@@ -163,7 +165,7 @@ public function logInAs($username) {
       }
       else {
         $alias = $this->_getConfig('alias') ? $this->_getConfig('alias') . ' ' : '';
-        $output = Drush::runDrush($alias. 'uli --name=' . $username, $this->_getConfig('drush'), $this->_getConfig('working_directory'));
+        $output = Drush::runDrush($alias. '--uri=' . $this->_getConfig('site') . ' uli --name=' . $username, $this->_getConfig('drush'), $this->_getConfig('working_directory'));
```
Comment (minorOffense) [inline src/Codeception/Module/DrupalUser.php:169]: Is this to add support for multisite?

**Review by minorOffense — CHANGES_REQUESTED** [review]

File: src/Codeception/Module/DrupalUser.php (hunk 2 above)
Comment (adubovskoy) [reply src/Codeception/Module/DrupalUser.php:169]: Yes, correct.

File: src/Codeception/Module/DrupalUser.php (hunk 1 above)
Comment (adubovskoy) [reply src/Codeception/Module/DrupalUser.php]: Ohh, thank you for noticing that!

**Review by adubovskoy — COMMENTED** [review]
Fixes added, thank you!

**Review by minorOffense — APPROVED** [review]

**Pull request merged**